spark/agent/src/main/scala/za/co/absa/spline/harvester/conf/DefaultSplineConfigurer.scala

```diff
@@ -41,8 +42,21 @@ object DefaultSplineConfigurer {
       */
     val MODE = "spline.mode"
     val MODE_DEFAULT = BEST_EFFORT.toString
-  }
 
+    /**
+     * Which lineage dispatcher should be used to report lineages (to Spline or elsewhere)
+     */
+    val LINEAGE_DISPATCHER_CLASS = "spline.lineage_dispatcher.className"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'LINEAGE_DISPATCHER_CLASS' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4506530)